cmd/vcfFilter/vcfFilter_test.go

```diff
@@ -66,16 +72,18 @@ func TestVcfFilter(t *testing.T) {
 			refStrongAltWeakOnly:           v.refStrongAltWeakOnly,
 			notRefStrongAltWeak:            v.notRefStrongAltWeak,
 			notRefWeakAltStrong:            v.notRefWeakAltStrong,
-			id:                             v.id, //raven's note: added id
+			id:                             v.id,
+			subSet:                         v.subSet,
 		}
 
-		vcfFilter(v.inputFile, "tmp.vcf", c, v.groupFile, false, false)
-		records, _ := vcf.Read("tmp.vcf")
+		vcfFilter(v.inputFile, v.tmpOutFile, c, v.groupFile, false, false, v.randSeed, v.setSeed)
+		records, _ := vcf.Read(v.tmpOutFile)
 		expected, _ := vcf.Read(v.expectedOutputFile)
 		if !vcf.AllEqual(records, expected) {
 			t.Errorf("Error in vcfFilter.")
+		} else {
```

Comment:
That is a nice bit of logic so that we keep the temp file around when it does not pass the test.

Reply:
yes we should make this standard practice. Maybe worth a note in code review